### ISSUE TITLE
composeRoot not found error in case of cancelled compose with at least one recipient

### ISSUE THREAD

**Issue by kinkoazc** [KartikTalwar/gmail.js#756]
https://github.com/KartikTalwar/gmail.js/blob/edd1beeacf9faa9f82af7929706e95d0cd83e463/src/gmail.js#L2529

**Comment by josteink** [comment]
I'm not sure I fully understand the scope of this bug.

From a API usage point of view, are there use-cases which does not work as expected?

**Comment by josteink** [comment]
In that case, I guess "error" is more severe than it needs to be. But it could be because of something unexpected happening too.

How about warning instead of error?

**Comment by kinkoazc** [comment]
Yes, that is a good idea. But only for the cases other than compose close(not sure how this case can be identified among the others, but this would be the ideal behavior).

**Comment by josteink** [comment]
Feel free to give it a try. PRs welcome 🙂
